### PR TITLE
Create free-courses-th.md

### PR DESCRIPTION
**Pull request by siriwatw**
## Hacktoberfest notes:

- due to volume of submissions, we may not be able to review PRs that do not pass tests and do not have informative titles.
- please read our [contributing guidelines](/CONTRIBUTING.md)
- be sure to check the output of Travis-CI for linter errors
- if this is your first open source contribution, make sure it's not your last!

## What does this PR do?
Add Resource(s) 

### Checklist:
- [x] Not a duplicate
- [x] Included author(s) if appropriate
- [x] Lists are in alphabetical order
- [ ] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

**Comment by SethiPandi** [comment]
You only need 1 line break under the first line, and you only need 2 line breaks above each new section or subsection.

Could you please update your PR so the line breaks /spacing is standardised?

You can check out the contribution guidelines (CONTRIBUTE.md) for more information on formatting and other guidelines for this repository. You can click on the failed check below and head to Travis CI to see exactly what was wrong with your PR and how to fix it!

File: free-courses-th.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  * [Vue.js](#vuejs)
+* [Python](#python)
+
+
+### C
+
+* [บทเรียนภาษา C](http://marcuscode.com/lang/c) (MarcusCode)
+
+
+
```
Comment (SethiPandi) [inline free-courses-th.md]: Remove one line break from here, please. There should only be 2!

File: free-courses-th.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+### C-Sharp
+
+* [บทเรียนภาษา C#](http://marcuscode.com/lang/csharp) (MarcusCode)
+
+
+
```
Comment (SethiPandi) [inline free-courses-th.md]: Remove one line break from here, please. There should only be 2!

File: free-courses-th.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+### CPP
+
+* [บทเรียนภาษา C++](http://marcuscode.com/lang/cpp) (MarcusCode)
+
+
+
```
Comment (SethiPandi) [inline free-courses-th.md]: Remove one line break from here, please. There should only be 2!

File: free-courses-th.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+### Git
+
+* [มาเรียนรู้ Git แบบง่ายๆกันเถอะ](https://blog.nextzy.me/มาเรียนรู้-git-แบบง่ายๆกันเถอะ-427398e62f82)  (Ake Exorcist)
+* [สอน git และ github เบื้องต้น](https://www.youtube.com/playlist?list=PLoTScYm9O0GGsV1ZAyP4m_iyAbflQrKrX)  (prasertcbs)
+* [สอนใช้ Git - Version Control System](https://www.youtube.com/playlist?list=PLjPfp4Ph3gBrgVPZySWHZwxXSxdgOKhQ-)  (CMDev)
+
+
+
```
Comment (SethiPandi) [inline free-courses-th.md]: Remove one line break from here, please. There should only be 2!

File: free-courses-th.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+#### React
+* [สอน React.JS Tutorial](https://www.youtube.com/playlist?list=PLjPfp4Ph3gBo5SmWJXwv4oKDfeTXA7xgw)  (CMDev)
+
+
+#### Vue.js
+* [สอนเขียน VueJS](https://www.youtube.com/playlist?list=PLjPfp4Ph3gBry3sJDNrbqor5ikjwGDJ_7)  (CMDev)
+
+
+
```
Comment (SethiPandi) [inline free-courses-th.md]: Remove one line break from here, please. There should only be 2!

File: free-courses-th.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
+### Index
+
+
```
Comment (SethiPandi) [inline free-courses-th.md]: Remove one line break from here, please. There should only be **1**!

**Review by SethiPandi — CHANGES_REQUESTED** [review]
~~I've dropped a few comments which is more clear on which lines should be removed!~~

Apparently, I reviewed the first version. :thinking: 
I'll do that again, sorry.

File: free-courses-th.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+* [บทเรียนภาษา Javascript](http://marcuscode.com/lang/javascript) (MarcusCode)
+
+
+#### Angular.js
+
+* [มือใหม่หัดใช้ Angular](https://priefydev.wordpress.com/tag/angular/) (Priefy Dev.)
+
+
+#### NodeJS
+* [สอน Node.js เบื้องต้น](https://www.youtube.com/playlist?list=PLoTScYm9O0GERtEdsPHK5Q-cdor5ADnyM)  (pracertcbs)
```
Comment (SethiPandi) [inline free-courses-th.md:50]: Please add a line break between line 49 and 50.

File: free-courses-th.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+* [สอน Nodejs เบื้องต้น สำหรับผู้เริ่มต้นศึกษา Nodejs](https://www.youtube.com/playlist?list=PLEE74DyIkwEkWkVWy3TbjrTICVF_eUdyc)  (Kong Ruksiam)
+
+
+#### React
+* [สอน React.JS Tutorial](https://www.youtube.com/playlist?list=PLjPfp4Ph3gBo5SmWJXwv4oKDfeTXA7xgw)  (CMDev)
```
Comment (SethiPandi) [inline free-courses-th.md:55]: Please add a line break between line 54 and 55.

File: free-courses-th.md [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+
+
+#### Vue.js
+* [สอนเขียน VueJS](https://www.youtube.com/playlist?list=PLjPfp4Ph3gBry3sJDNrbqor5ikjwGDJ_7)  (CMDev)
```
Comment (SethiPandi) [inline free-courses-th.md:59]: Please add a line break between line 58 and 59.

File: free-courses-th.md [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+* [มาเรียนรู้ Git แบบง่ายๆกันเถอะ](https://blog.nextzy.me/มาเรียนรู้-git-แบบง่ายๆกันเถอะ-427398e62f82)  (Ake Exorcist)
+* [สอน git และ github เบื้องต้น](https://www.youtube.com/playlist?list=PLoTScYm9O0GGsV1ZAyP4m_iyAbflQrKrX)  (prasertcbs)
+* [สอนใช้ Git - Version Control System](https://www.youtube.com/playlist?list=PLjPfp4Ph3gBrgVPZySWHZwxXSxdgOKhQ-)  (CMDev)
+
+
+### JavaScript
+
+* [จาวาสคริปต์เบื้องต้น](https://phyblas.hinaboshi.com/saraban/javascript) (Phyblas)
+* [สอน JavaScript](https://www.youtube.com/playlist?list=PL_xSQKvnccplgKmdtqizMGRh11witheTM)  (Zinglecode)
+* [บทเรียนภาษา Javascript](http://marcuscode.com/lang/javascript) (MarcusCode)
```
Comment (SethiPandi) [inline free-courses-th.md:41]: On the website the resource is called: `ภาษา JavaScript` on the website.
Could we please use the name as described there?

**Review by SethiPandi — CHANGES_REQUESTED** [review]
I've left some comments on the correct place this time, not sure how I did the other one before.

Could you please also ensure all lists are in alphabetical order!

File: free-courses-th.md (hunk 10 above)
Comment (SethiPandi) [reply free-courses-th.md]: Could you please write `JavaScript` with a capital S, the same way that it is on the website?
Java**S**cript is the correct spelling for it anyway.

**Comment by eshellman** [comment]
Wow, nice effort!
Thanks, and welcome to free-programming-books!

**Comment by eshellman** [comment]
Now you need to add a link from README.md (I should have flagged that before merging.)

**Pull request merged**